### ISSUE TITLE
Tracker is no more installing on newest version of Hum Hub

### ISSUE THREAD

**Issue by lisandi** [githubjeka/tracker-issues#47]
Hi we tried to install Tracker today from the repo in HumHub and it won't install at all!

**Comment by wehowski** [comment]
Yes, after update it does not work anymore: [https://github.com/humhub/humhub/issues/5986#issuecomment-1362581236](https://github.com/humhub/humhub/issues/5986#issuecomment-1362581236) !?!
